lib/client.js

```diff
@@ -14,6 +14,7 @@ var HttpClient = require('./http'),
   findPrefix = require('./utils').findPrefix,
   _ = require('lodash'),
   concatStream = require('concat-stream'),
+  Promise = require("bluebird"),
```

Comment:
could you please change the name from `Promise` to `BPromise` or something the like so that we don't collide with the **native** `Promise` class. 👍  (in `soap.js` too)